frontend/src/components/SecretRow.vue

```diff
@@ -5,112 +5,113 @@ SPDX-License-Identifier: Apache-2.0
-              <v-icon class="error--text">mdi-delete</v-icon>
-            </v-btn>
-          </div>
-        </template>
-        <span v-if="!isOwnSecret">You can only delete secrets that are owned by you</span>
-        <span v-else-if="relatedShootCount > 0">You can only delete secrets that are currently unused</span>
-        <span v-else>Delete Secret</span>
-      </v-tooltip>
-    </v-list-item-action>
-
-    <v-list-item-action>
-      <v-tooltip top>
-        <template v-slot:activator="{ on }">
-          <div v-on="on">
-            <v-btn :disabled="!isOwnSecret" icon @click.native.stop="onUpdate">
-              <v-icon class="action-button--text">mdi-pencil</v-icon>
-            </v-btn>
-          </div>
-        </template>
-        <span v-if="!isOwnSecret">You can only edit secrets that are owned by you</span>
-        <span v-else>Edit Secret</span>
-      </v-tooltip>
-    </v-list-item-action>
-  </v-list-item>
+      </div>
+    </td>
+    <td v-if="selectedHeaders.infrastructure">
+      <vendor extended :cloud-provider-kind="item.infrastructureName" :cloud-profile-name="item.cloudProfileName"></vendor>
+    </td>
+    <td v-if="selectedHeaders.details">
+      <v-list color="transparent">
+        <v-list-item class="pa-0">
+          <v-list-item-content class="pa-0">
+            <v-list-item-subtitle>
+              <span v-for="({ label }, index) in item.details" :key="label">
+                <span>{{label}}</span>
+                <span v-if="index !== item.details.length - 1"> / </span>
+              </span>
+            </v-list-item-subtitle>
+            <v-list-item-title>
+              <span v-for="({ value }, index) in item.details" :key="value">
+                <span v-if="value">{{value}}</span>
+                <span v-else class="font-weight-light text--disabled">unknown</span>
+                <span v-if="index !== item.details.length - 1"> / </span>
+              </span>
+            </v-list-item-title>
+          </v-list-item-content>
+        </v-list-item>
+      </v-list>
+    </td>
+    <td v-if="selectedHeaders.relatedShootCount">
+      <div class="d-flex">
+        {{item.relatedShootCountLabel}}
+      </div>
+    </td>
+    <td v-if="selectedHeaders.actions">
+      <div class="d-flex flex-row justify-end">
+        <v-tooltip top v-if="canPatchSecrets">
+          <template v-slot:activator="{ on }">
+            <div v-on="on">
+              <v-btn :disabled="!item.isOwnSecret || !item.isSupportedInfrastructure" icon @click="onUpdate">
+                <v-icon class="action-button--text">mdi-pencil</v-icon>
+              </v-btn>
+            </div>
+          </template>
+          <span v-if="!item.isOwnSecret">You can only edit secrets that are owned by you</span>
+          <span v-else-if="!item.isSupportedInfrastructure">This infrastructure type is currently not supported by the dashboard</span>
```

Comment:
```js
          <span v-else-if="!item.isSupportedInfrastructure">This infrastructure type is currently not supported by the dashboard</span>
```
this is not a limitation of the dashboard itself but rather a configuration issue, or not?

Reply:
If an extension for an infrastructure kind is installed on a landscape that the Dashboard is not aware of / has not yet implemented, then secrets (and also clusters) of this infrastructure still appear in the dashboard. It happened on our dev landscapes, for instance. While viewing cluster details works fine (most of the time), editing such a secret is not possible as we do not know which fields we need to maintain.

Reply:
```js
isSupportedInfrastructure: includes(this.sortedCloudProviderKindList, secret.metadata.cloudProviderKind)
```
nevermind, I confused `sortedCloudProviderKindList` with the `cloudProfiles`